### PR TITLE
Add tooltip for cancel button in panel

### PR DESCRIPTION
**Pull request by shivasai09**
#### Description of changes

added the title attribute for the cancel icon in the panel which is same as aira-label

**after fix**
tooltip

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8842)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx [hunk 1]
```diff
@@ -277,6 +277,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
           className={this._classNames.closeButton}
           onClick={this._onPanelClick}
           ariaLabel={closeButtonAriaLabel}
+          title={closeButtonAriaLabel}
```
Comment (KevinTCoughlin) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx]: I'm uncertain we should do this by default. I think the guidance from W3 ARIA is to not use both `aria-label` and `title`. Perhaps we can make this opt-in. 

https://github.com/nvaccess/nvda/issues/7841#issuecomment-358644281

>The last resort is to use text from a tooltip attribute (such as the title attribute in HTML). This is used only if nothing else, including subtree content, has provided results.

Having both may cause assistive technology to double-read.

Comment (shivasai09) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx]: >Having both may cause assistive technology to double-read. I haven't tried this evening admittedly.

Hey @KevinTCoughlin  i have tried it, screen reader is not reading it double time!

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx]: Good to hear 👍. As long as Keros doesn't flag it as a violation then we're good. We might want to add a snapshot test also.

I think Keros as part of CI is currently disabled

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx]: I tested this locally in Edge + Narrator and @shivasai09 is correct, the attribute does not result in a double read.

**Comment by shivasai09** [comment]
any update?

**Pull request merged**